build.sbt

```diff
@@ -3,22 +3,25 @@ com.typesafe.sbt.SbtGit.versionWithGit
 
 organization := "com.swoop"
 
-bintrayOrganization := Some("swoop-inc")
-bintrayRepository := "spark-records"
-bintrayPackageLabels := Seq("apache", "spark", "apache-spark", "scala", "big-data", "spark-records", "dataset", "swoop")
-resolvers += "swoop-bintray" at "https://dl.bintray.com/swoop-inc/maven/"
 version := "3.0.0"
 
-licenses +=("Apache-2.0", url("http://apache.org/licenses/LICENSE-2.0"))
-
 val vSpark = "3.0.1"
 
+credentials += Credentials(Path.userHome / ".sbt" / "sonatype_credentials")
```

Comment:
Is there a way to make this path configurable in the command line? It would be useful for CI systems.

Reply:
@greglu - [sbt-ci-release](https://github.com/olafurpg/sbt-ci-release), the best way to setup open source SBT projects, requires that these credentials are not set (they're defined as GitHub secrets).  [spark-excel](https://github.com/crealytics/spark-excel) is a good example of a project that uses sbt-ci-release.  Check out [the release file](https://github.com/crealytics/spark-excel/blob/main/.github/workflows/release.yml).

Reply:
All right sounds good :+1: